cmd/signature-v4_test.go

```diff
@@ -54,14 +54,7 @@ func TestDoesPolicySignatureMatch(t *testing.T) {
 			},
 			expected: ErrInvalidAccessKeyID,
 		},
-		// (2) It should fail if the region is invalid.
-		{
-			form: http.Header{
-				"X-Amz-Credential": []string{fmt.Sprintf(credentialTemplate, accessKey, now.Format(yyyymmdd), "invalidregion")},
```

Comment:
This was failing with ErrSignatureDoesNotMatch and but this test case was not looking like it was written for ErrSignatureDoesNotMatch. Hence it would cause confusion to keep it.

There is another test case for ErrSignatureDoesNotMatch.